corehq/apps/data_interfaces/tests/test_auto_case_updates.py

```diff
@@ -10,7 +10,8 @@
 
 from corehq.form_processor.backends.sql.dbaccessors import CaseAccessorSQL
 from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
-from corehq.form_processor.tests.utils import run_with_all_backends, FormProcessorTestUtils, set_case_property_directly
+from corehq.form_processor.tests.utils import (run_with_all_backends, FormProcessorTestUtils,
+    set_case_property_directly, set_parent_case as set_actual_parent_case)
```

Comment:
thanks @millerdev for noticing, looks like the data_interfaces tests have been excluded from the test runner. this pr fixes that and fixes the import error: https://github.com/dimagi/commcare-hq/pull/11567